src/Components/Dropdowns/Composed/TypeAheadDropDown.tsx

```diff
@@ -126,82 +112,56 @@ export const TypeAheadDropDown: FC<OwnProps> = ({
     } else {
       setQueryResults(items)
     }
-  }, [items, typedValue])
-
-  /**
-   *  using a ref to hold an instance variable:  what was last typed,
-   *  because without this 'click to select' doesn't work.
-   *  (you click to select, which clicks out of the dropdown, so then the dropdown sets
-   * the text to what was last selected.  this works fine for a class component,
-   * but here it uses the stale state of what was previously selected.
-   * this way, what was selected is saved in the ref.)
-   */
```

Comment:
I didn't find this to be true, the dropdown works without needing to store a ref? Am I missing something? 

https://user-images.githubusercontent.com/18511823/182245883-111cc654-6ee4-4db8-84ae-bf8588ca0e7a.mov

Reply:
I don't think you're missing anything: I see that this is working.  But do you know what the source of the rendering delay is here? It looks like there's a significant gap between the user selecting an option, and it registering in the typeahead.

Reply:
That delay is just the dashboard changing the variable in all the places, not with the typeahead dropdown 👍